pages/book/[isbn].tsx

```diff
@@ -0,0 +1,145 @@
+import { Badge, Box, Button, Flex, Grid, HStack, Spacer, Text, Wrap } from "@chakra-ui/react";
+import Image from 'next/image'
+import { PostCardList } from "@components/CardList";
+import Layout from "@components/Layout";
+import { useBookQuery } from "@lib/hooks/book";
+import type { NextPage } from "next";
+import { useRouter } from "next/router";
+import Link from "next/link";
+import { resolveImageUrl } from "@lib/helpers/frontend/resolve-image-url";
+
+const BookPage: NextPage = () => {
```

Comment:
I think you should add the icons to the different fields and the badges like in the Figma. All the icons should be present in the heroicons icon pack, except the barcode/ISBN one. I have incorporated the ISBN one in my PR, so once that's merged, we can use that here.